apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -396,27 +404,36 @@ export default function Success(props: PageProps) {
                 {!isFeedbackMode && (
                   <>
                     <div
-                      className={classNames(
-                        "mx-auto flex items-center justify-center",
-                        !giphyImage && !isCancelled && !needsConfirmation
-                          ? "bg-success h-12 w-12 rounded-full"
-                          : "",
-                        !giphyImage && !isCancelled && needsConfirmation
-                          ? "bg-subtle h-12 w-12 rounded-full"
-                          : "",
-                        isCancelled ? "bg-error h-12 w-12 rounded-full " : ""
-                      )}>
-                      {giphyImage && !needsConfirmation && !isCancelled && (
-                        // eslint-disable-next-line @next/next/no-img-element
-                        <img src={giphyImage} alt="Gif from Giphy" />
-                      )}
-                      {!giphyImage && !needsConfirmation && !isCancelled && (
-                        <Icon name="check" className="h-5 w-5 text-green-600 dark:text-green-400" />
+                      className={classNames(isRoundRobin && "min-w-32 min-h-24 relative mx-auto h-24 w-32")}>
+                      {isRoundRobin && bookingInfo.user && (
+                        <Avatar
+                          className="mx-auto flex items-center justify-center"
+                          alt={bookingInfo.user.name || bookingInfo.user.email}
+                          size="xl"
+                          imageSrc={`${bookingInfo.user.avatarUrl}`}
+                        />
                       )}
-                      {needsConfirmation && !isCancelled && (
-                        <Icon name="calendar" className="text-emphasis h-5 w-5" />
-                      )}
-                      {isCancelled && <Icon name="x" className="h-5 w-5 text-red-600 dark:text-red-200" />}
+                      <div
+                        className={classNames(
+                          "mx-auto flex h-12 w-12 items-center justify-center rounded-full",
+                          isRoundRobin &&
+                            "absolute bottom-0 right-0 z-10 h-12 w-12 border-8 border-[--cal-bg] dark:border-[--cal-bg-muted]",
```

Comment:
> border-[--cal-bg] dark:border-[--cal-bg-muted]

this is not how we use variables in the codebase. try using border-subtle , border-muted etc.

Reply:
Made changes to border styles to include cal bg and bg-muted